apps/condo/domains/common/utils/helpers.ts

```diff
@@ -91,3 +96,30 @@ export const getId = (record: IRecordWithId): string | null => get(record, 'id',
  * @param key
  */
 export const getFilteredValue = <T>(filters: T, key: string | Array<string>): FilterValue => get(filters, key, null)
+
+type humanizeDaysType = (days: number) => string
+/**
+ * Formats days into a human readable string
+ * @param days
+ */
+export const humanizeDays: humanizeDaysType = (days) => {
```

Comment:
You can't hardcode the locales

Reply:
but it's really needed - ready-made methods don't work as they should

Reply:
Made abbreviations for days, now this function is not needed